source/best-practices/file-organization.rst

```diff
@@ -0,0 +1,159 @@
+文件管理
+========
+
+:本节贡献者: `田冬冬 <https://me.seisman.info>`__ （作者）、`姚家园 <https://github.com/core-man>`__ （作者）
+:最近更新日期: 2021-01-12
+:预计阅读时间: X 分钟
+
+正确管理电脑文件是非常重要的。错误的文件管理方式会使你花费更多的时间去寻找文件，
+拖慢你的科研进度，还可能导致文件的大量冗余与误删。
+这一节，我们总结了自己在科研工作中关于文件组织与管理的经验，供读者参考。
+
+本文主要面向 Linux 和 macOS 用户，下面提到的\ **家目录**\ 在 Linux 下指的是 :file:`/home/用户名`，
+在 macOS 下指的是 :file:`/Users/用户名`。
+
+基本原则
+--------
+
+本文的文件组织与管理遵循如下几条基本原则：
+
+1. 保持家目录干净整洁
+2. 所有科研相关的文档、代码、图片等均放在家目录的子目录下
+3. 家目录下的每个子目录都有特定的用途，用于放置特定的文件
+
+目录结构
+--------
+
+根据以上原则，我们推荐在家目录下建立如下子目录：
+
+-   Linux 或 macOS 系统在安装后自动创建的常见目录：
+
+    - :file:`Documents`
+    - :file:`Downloads`
+    - :file:`Videos`
+    - :file:`Music`
+    - :file:`Desktop`
+
+-   为了满足日常科研需求，在家目录下新建的、用于存储科研相关文件的目录
+
+    - :file:`bin`
+    - :file:`src`
+    - :file:`software`
+    - :file:`opt`
+    - :file:`projects`
+    - :file:`codes`
+    - :file:`data`
+    - :file:`workspace`
```

Comment:
We may have a better order and the following will also follow this order.

My suggestion is:

```suggestion
    - :file:`software`
    - :file:`src`
    - :file:`codes`
    - :file:`opt`
    - :file:`bin`
    - :file:`data`
    - :file:`projects`
    - :file:`workspace`
```

Reply:
Looks good to me.